mix.exs

```diff
@@ -46,7 +46,7 @@ defmodule Phoenix.LiveDashboard.MixProject do
       {:telemetry_metrics, "~> 0.6.0"},
       {:ecto_psql_extras, "~> 0.7", optional: true},
       {:ecto_mysql_extras, "~> 0.5", optional: true},
-      {:ecto, "~> 3.6.2 or ~> 3.7", optional: true},
+      {:ecto, "~> 3.6.2 or ~> 3.7 or ~> 3.8", optional: true},
```

Comment:
~> 3.7 allows 3.7.x, 3.8.x, 3.9.x, etc. So this should not be necessary.

Reply:
🤦 My apologies, I found this when investigating but as I'd used override already I didn't think twice.